modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/SupportExportObjectEntriesAtInstancelevel.testcase

```diff
@@ -0,0 +1,329 @@
+
+			FileUtil.replaceStringInFile(${filePath}, 39649, ${accountEntryId});
+
+			FileUtil.replaceStringInFile(${filePath}, "66ff0575-feb9-38fc-ae9d-a5fec2248b23", ${accountEntryERC});
+
+			ImportExport.assertFileContent(
+				comparisonFileName = "json_objectEntry_export_all_fields.txt",
+				fileName = "export.json");
+		}
+	}
+
+	@priority = 4
+	test CanExportSpecificFieldsWithJSONTConfigurationNode {
+		task ("And Given 'C_Student (v1_0 - Liferay Object REST)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+		}
+
+		task ("When exporting file in JSONT format with field 'name' selected") {
+			ImportExport.configureExport(
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				exportFields = "name",
+				exportFileFormat = "JSONT");
+
+			Button.click(button = "Export");
+
+			WaitForElementPresent(locator1 = "ImportExport#EXECUTION_SUCCESS");
+		}
+
+		task ("Then 'name' value for created entry, actions node, configuration node available in the dowloaded file") {
+			ImportExport.unzipDownloadedExportFile();
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
+				expectedValue = "Able",
+				fileName = "export.batch-engine-data.json",
+				jsonObject = "$.items..name");
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
+				expectedValue = "/o/headless-batch-engine/v1.0/import-task/com.liferay.object.rest.dto.v1_0.ObjectEntry",
+				fileName = "export.batch-engine-data.json",
+				jsonObject = "$.actions.createBatch.href");
+
+			BatchEngine.assertExportedFileContainsCorrectObject(
+				expectedValue = "com.liferay.object.rest.dto.v1_0.ObjectEntry",
+				fileName = "export.batch-engine-data.json",
+				jsonObject = "$.configuration.className");
+		}
+	}
+
+	@priority = 4
+	test CanExportSystemObjectItems {
+		property portal.acceptance = "true";
+
+		task ("And Given 'Account (v1_0 - Liferay Headless Admin User)' as Entity Type in Data Migration Center --> Export File") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.selectEntity(entityType = "Account (v1_0 - Liferay Headless Admin User)");
+		}
+
+		task ("When exporting file in JSONT format with all fields selected") {
+			ImportExport.configureExport(
+				entityType = "Account (v1_0 - Liferay Headless Admin User)",
+				exportFields = "Attribute Code",
+				exportFileFormat = "JSONT");
+
+			Button.click(button = "Export");
+
+			WaitForElementPresent(locator1 = "ImportExport#EXECUTION_SUCCESS");
+		}
+
+		task ("Then all items fields values, actions node, configuration node available in the dowloaded file") {
+			ImportExport.unzipDownloadedExportFile();
+
+			var response = AccountAPI.getAccounts();
+
+			var dateCreated = JSONPathUtil.getProperty(
```

Comment:
Can we just get all $.items instead of every single node?

Reply:
I tested my local, and the get response and the export files have different sorts and formats, so it can not use get all $.items instead of every single node

Reply:
I used  the "for" function to get all the values